### DISCUSSION TITLE
Is detailed energy monitoring possible?

### DISCUSSION THREAD

**Discussion by HisDudeness3008** [arendst/Tasmota#17347] [Q&A]
Hello everyone! I'm waiting for a good price on a [Gosund EP2](https://de.gosund.com/products/gosund%E2%84%A2-wlan-smart-steckdose-ep2-2-pack?variant=3bbcde92-f7bf-4d61-9b6d-41cedb4d1ded), due to their small size, and I am interested in tasmotising them. I have a couple of doubts about the possibilities Tasmota will give, however.

1. Is it possible to have a more detailed energy monitoring? For example, to have energy consumption in kWh hourly or at least more times in a day, or to get an history of power absorption in W instead of just the instant data and history of energy alone? Is it something hardware specific?
2. This model has a rather bright LED I'm not sure whether can be disabled or not from the stock app. Is it possible for Tasmota to turn it off?

Thanks!

**Comment by sfromis** [comment]
Tasmota does not have features for any sort of detailed history, like collecting hourly data back in time. Such things are expected to be done backend instead of on the device itself, as the web UI is not mean as a "dashboard" for daily operation.

You have a number of ways to amend how the status leds are handled. https://tasmota.github.io/docs/Lights/#status-leds

https://templates.blakadder.com/gosund_EP2.html has two, you can also reconfigure (or even unconfigure) pins if you do not want the status led function at all.
